packages/emails/biome.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
+    "extends": ["../../biome.json"],
+    "overrides": [
+      {
+        "include": ["*.ts", "*.tsx"],
+        "javascript": {
+          "formatter": {
+            "quoteStyle": "single"
+          },
+          "globals": [
```

Comment:
Needed for unit test

Reply:
yeah, it does not pass without it. 
In fact, I just saw you created the same biome conf for the e2e package, so i ended up copy paste it